ea_airflow_util/callables/file_type.py

```diff
@@ -0,0 +1,48 @@
+import json
+import xmltodict
+import os 
+import logging
+    
+def xml_to_json(
+    xml_path: str,
+    output_path: str = None
+):
+    """
+    Transform an XML file into a JSON format.
+    """    
+    
+    # Open the input XML file and read data in form of python dictionary using the "xmltodict" module.
+    try:
+        with open(xml_path) as xml_file:
+            data_dict = xmltodict.parse(xml_file.read())
+    except FileNotFoundError as error:
+        logging.error(f"Error: {str(error)} (XML file not found)")
+    except Exception as error:
+        logging.error(f"Error: {str(error)}")
+      
+    # Generate the json_data object using json.dumps().
+    json_data = json.dumps(data_dict)
+
+    # Check if output_path is provided, otherwise set it to a default value (original XML folder path).
+    if output_path is None:
+        output_path = os.path.dirname(xml_path)
+        output_directory = f'{output_path}/json'
+    else:
+        output_directory = f'{output_path}/json'
```

Comment:
If an output path is explicitly provided, will we still need to add the JSON subfolder?